### PR TITLE
bnfc 2.8.3 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This formula has an inevitable audit failure because of trailing whitespace.  The trailing whitespace is _very intentional_ and the tests would break if it wasn't there.

EDIT:  removed the audit failure the "blunt" way

### CODE REVIEW

File: Formula/bnfc.rb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  homepage "https://bnfc.digitalgrammars.com/"
+  url "https://github.com/BNFC/bnfc/archive/v2.8.3.tar.gz"
+  sha256 "ba0b6ab36954a0891b4ad3125cefdd6d441d2c73d174cd8eff344e68ae2fd203"
+  head "https://github.com/BNFC/bnfc.git"
+
+  depends_on "cabal-install" => [:build, :test]
+  depends_on "ghc" => [:build, :test]
+  depends_on "sphinx-doc" => :build
+  depends_on "antlr" => :test
+  depends_on :java => :test
```
Comment (SMillerDev) [inline Formula/bnfc.rb]:
```suggestion
  depends_on "openjdk" => :test
```

**Comment by SMillerDev** [comment]
Brew audit fails on the styling

File: Formula/bnfc.rb (hunk 1 above)
Comment (pthariensflame) [reply Formula/bnfc.rb]: Done!

**Comment by pthariensflame** [comment]
@SMillerDev Does that look acceptable now?

**Review by SMillerDev — APPROVED** [review]

**Comment by issyl0** [comment]
Thank you for this, @pthariensflame!

**Pull request closed without merging**